rest/route/annotations_test.go

```diff
@@ -344,8 +344,13 @@ func TestAnnotationByTaskPutHandlerParse(t *testing.T) {
 	assert.NoError(t, db.ClearCollections(annotations.Collection, task.Collection, task.OldCollection))
 	tasks := []task.Task{
 		{Id: "t1", Execution: 1, Status: evergreen.TaskFailed},
-		{Id: "t2", Execution: 1, Status: evergreen.TaskFailed},
-		{Id: "t3", Execution: 1, Status: evergreen.TaskSucceeded},
+		{Id: "t2", Execution: 1, Status: evergreen.TaskSystemUnresponse},
+		{Id: "t3", Execution: 0, Status: evergreen.TaskSystemFailed},
+		{Id: "t4", Execution: 0, Status: evergreen.TaskTimedOut},
+		{Id: "t5", Execution: 0, Status: evergreen.TaskSetupFailed},
+		{Id: "t6", Execution: 0, Status: evergreen.TaskSucceeded},
+		{Id: "t7", Execution: 0, Status: evergreen.TaskWillRun},
```

Comment:
I don't think we should let someone create a task annotation for TaskWillRun. I think we should only allow it for tasks that have already finished and are in some kind of failure status.

Reply:
haha I commented to quickly. You are checking that it's NOT this. this is perfect.